Objects/genericaliasobject.c

```diff
@@ -697,7 +697,7 @@ static PyTypeObject Py_GenericAliasIterType = {
 
 static PyObject *
 ga_iter(PyObject *self) {
-    gaiterobject *gi = PyObject_GC_New(gaiterobject, &Py_GenericAliasIterType);
+    gaiterobject *gi = PyObject_GC_New(gaiterobject, &_Py_GenericAliasIterType);
```

Comment:
I'm surprised that it works to use a static type without calling PyType_Ready() on it! It looks like a bug (fixed by this PR thanks to _PyTypes_InitTypes()).